compiled_starters/csharp/src/Program.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.IO;
+
+static bool MatchPattern(string inputLine, string pattern) {
+    if (pattern.Length == 1) {
+        return inputLine.Contains(pattern);
+    } else {
+        throw new ArgumentException($"Unhandled pattern: {pattern}");
+    }
+}
+
+// You can use print statements as follows for debugging, they'll be visible when running tests.
+Console.WriteLine("Logs from your program will appear here!");
+
+// Uncomment this block to pass the first stage
+//
+// if (args[0] != "-E") {
+//     Console.WriteLine("Expected first argument to be '-E'");
+//     Environment.Exit(2);
+// }
+// string pattern = args[1];
+// string inputLine = Console.In.ReadToEnd();
+// if (MatchPattern(inputLine, pattern)) {
```

Comment:
@ryan-gang let's make this even smaller - can we do just the last 5 lines? That'd be similar to what we do for Python